### PR TITLE
LPS-177827 [CP 2.0] Create Replacement Activation Key Request Form

### PR DESCRIPTION
**Pull request by JoMaAlves**

### CODE REVIEW

**Comment by JoMaAlves** [comment]
ci:test:relevant

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/src/main/resources/site-initializer/layouts/04_replacement-activation-key-form/page-definition.json [hunk 1]
```diff
@@ -0,0 +1,540 @@
...
+									},
+									"indexed": true
+								},
+								"type": "Fragment"
+							},
+							{
+								"definition": {
+									"fragment": {
+										"key": "custom-dialect-text",
+										"siteKey": "CP"
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/src/main/resources/site-initializer/layouts/04_replacement-activation-key-form/page-definition.json:474]:
```suggestion
										"siteKey": [$GROUP_KEY$]
```

**Review by jairmedeiros — CHANGES_REQUESTED** [review]

**Comment by JoMaAlves** [comment]
ci:forward

**Comment by kevenleone** [comment]
Just starting reviewing :)

**Pull request closed without merging**